### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by manfredlotz** [Dherse/codly#113]
It seems that codly doesn't support accessibility offered by Typst 0.14.0

If I compile this

````typst
#import "@preview/codly:1.3.0": *
#show: codly-init.with()


#set document(title: "Sample document")

= This is codly

#codly(
  languages: (
    rust: (name: "Rust", icon: "🦀", color: rgb("#CE412B")),
  )
)


```rust
pub fn main() {
    println!("Hello, world!");
}
```
````

I get 

```
error: PDF/UA-1 error: missing alt text
     ┌─ @preview/codly:1.3.0/src/lib.typ:1807:2
     │
1807 │ ╭   figure(
1808 │ │     kind: "__codly-end-block",
1809 │ │     supplement: none,
1810 │ │     numbering: none,
     · │
1814 │ │     caption: none,
1815 │ │   )[]
     │ ╰─────^
     │
     = hint: make sure your images and equations have alt text
```

**Comment by Andrew15-5** [comment]
A simple code block is enough to trigger this:

````typ
#import "@preview/codly:1.3.0": *
#show: codly-init
```
code
```
````

**Comment by Dherse** [comment]
Fixed by [deadb8e](https://github.com/Dherse/codly/commit/deadb8e8df67e27c1099f58c2d729b7048128175)

**Comment by Dherse** [comment]
Also referencing PR: #141
